specs/Pills.md

```diff
@@ -0,0 +1,188 @@
+# [Pills]()
+
+## Background
+
+PIlls should be used when representing an input, as a way to filter content, or to represent an attribute.
+
+## Prior Art
+
+- [Evergreen](https://evergreen.segment.com/components/badge-and-pill/)
+- [Lightning](https://www.lightningdesignsystem.com/components/pills/)
+- [OpenUI Research]()
+
+## Sample Code
+
+```jsx
+<Pills>
+  <Pill>Item 1</Pill>
+  <Pill>Item 2</Pill>
+</Pills>
+```
+
+## Variants
+
+- Shape: `circular`, `rounded`
+- Size: `smaller`, `small`, `medium`
```

Comment:
Why only those variants? Does current design spec have only those?

Reply:
Yep, design spec only has that